ci_test/unit_tests/test_unit_subgrid_sum.py

```diff
@@ -19,7 +19,7 @@
 
 # Data
 np.random.seed(202201184)
-_num_samples = 17
+_num_samples = 32

```

Comment:
Does this need to change?

Reply:
Comment from @aj-prime "we don’t support non power of 2 batch sizes."